test/parallel/test-dgram-send-default-host.js

```diff
@@ -16,7 +16,7 @@ const received = [];
 client.on('listening', common.mustCall(() => {
   const port = client.address().port;
   client.send(toSend[0], 0, toSend[0].length, port);
-  client.send(toSend[1], port);
+  client.send(new Uint8Array([...toSend[1]]), port);
```

Comment:
This is missing test cases for `socket.send(u8a, offset, length, port)` and `socket.send(buffer, port)`.

Also, is `new Uint8Array([...toSend[1]])` here the same as `new Uint8Array(toSend[1])`?